storcli.py

```diff
@@ -62,10 +63,12 @@ def main(args):
 def handle_common_controller(response):
     (controller_index, baselabel) = get_basic_controller_info(response)
 
+    # Older boards don't have this sensor at all ("Temperature Sensor for ROC" : "Absent")
     # Split up string to not trigger CodeSpell issues
-    if 'ROC temperature(Degree Celc' + 'ius)' in response['HwCfg'].keys():
-        response['HwCfg']['ROC temperature(Degree Celsius)'] = response['HwCfg'].pop('ROC temperature(Degree Celc' + 'ius)')
-    add_metric('temperature', baselabel, int(response['HwCfg']['ROC temperature(Degree Celsius)']))
+    for key in ['ROC temperature(Degree Celc' + 'ius)', 'ROC temperature(Degree Celsius)']:
```

Comment:
The string concatenation in the first key in the list is redundant. I'm not even sure why it was originally written like that before your changes.

Reply:
I just left it as it was.  If you want me to remove it as part of this patch, and/or include a codespell whitelist, I can do.

Reply:
I can't see anything in the repo that is invoking CodeSpell, so presumably it was just done that way by the original author because he / she had CodeSpell setup in _their_ environment. IMHO, if there's no way to include overrides for false positives in the source itself (like how pylint, shellcheck etc do it), then it's really up to _user_ of CodeSpell to configure their environment correctly.

Reply:
OK, removed.

Reply:
Yes, we had codespell setup in the node_exporter repo. We should probably re-add some kind of codespell here.